dbeam-core/src/main/java/com/spotify/dbeam/avro/JdbcAvroSchema.java

```diff
@@ -193,13 +197,21 @@ private static SchemaBuilder.FieldAssembler<Schema> createAvroFields(
   private static SchemaBuilder.UnionAccumulator<SchemaBuilder.NullDefault<Schema>>
       setAvroColumnType(
           final int columnType,
-          final Integer arrayItemType,
+          final Array arrayInstance,
           final int precision,
           final String columnClassName,
+          final String columnTypeName,
           final boolean useLogicalTypes,
           final SchemaBuilder.BaseTypeBuilder<
                   SchemaBuilder.UnionAccumulator<SchemaBuilder.NullDefault<Schema>>>
-              field) {
+              field) throws SQLException {
+    if (arrayInstance != null) {
+      LOGGER.warn("{} column type {} columnClassName {} arrayBaseType {} arrayBaseTypeName",
+          columnType,
+          columnClassName,
+          arrayInstance.getBaseType(), arrayInstance.getBaseTypeName());
+    }
```

Comment:
If arrayInstance is null, won't this throw an exception later on? What's the point of logging a warning here?

Reply:
Perhaps out of scope for this particular change, but a few users have run into NPE when they have array typed column that has NULL value:
- https://spotify.slack.com/archives/C047H3XMT/p1739380831269689
- https://spotify.slack.com/archives/C0QC1CLGK/p1744680349573479

Reply:
Agreed, see new commits